### PR TITLE
Revert "Support torch.mean with dim=None (#3752)"

### PR DESCRIPTION
**Pull request by wonjoolee95**
This reverts commit d9a03dda2d1e3444f9166cf30f337ac255ec3a8c.

### CODE REVIEW

**Review by JackCaoG — APPROVED** [review]
Thanks!

**Comment by wonjoolee95** [comment]
Merging this now to unblock head.
